unity/Holo/Assets/Editor/AssetDirs.cs

```diff
@@ -8,17 +7,29 @@ public static class AssetDirs
      * Never ends with path delimiter (/ or \).
      *
      * Internal note: when changing this constant, adjust also directory creation
-     * code in LoadModel. */
-    public const string TempAssetsDir = "Assets/Temporary";
+     * code in DataPreparator. */
+    
+public const string TempAssetsDir = "Assets/Temporary";
+
+
+    public static void CreateDirectory(string path)

```

Comment:
Do you need this method? `Directory.CreateDirectory` does this for you

Reply:
Right, I didn't know that, fixed.